main.go

```diff
@@ -18,7 +18,9 @@ import (
 )
 
 var (
-	projectID = flag.String("project_id", "", "cloud project ID where the profile will be uploaded")
+	projectID = flag.String("project_id", "", "cloud project ID where the profile will be uploaded (Required)")
+	service   = flag.String("service_name", "uploaded-profiles", "name of service for uploaded profiles")
```

Comment:
Could you rename the flag to simply "service"? This is for consistency with the agent configurations of the profiler, e.g. see [here](https://cloud.google.com/profiler/docs/profiling-java#agent_configuration) where it's `-service`, not `-service_name`.

Reply:
Yep, I'll make that change.